docs/pages/config.mdx

```diff
@@ -121,3 +121,7 @@ The global configuration keys are all optional.
 
   - **`worldsFile`** a `string`: JSON file for the chain to `World` deploy address mapping.
     The default is `./worlds.json`.
+
+  - <a id="upgradeableWorldImplementation" />
+    **`upgradeableWorldImplementation`** a `bool`: Whether the `World` is to be deployed behind a proxy to [enable upgrades](/world/upgrade).
```

Comment:
Worlds are technically already upgradeable, but only via tables and systems.

We should clarify that this upgrades the _core implementation_ of a world, the things that can't be upgraded through tables/systems, and the things that may get security patches in the future.